\doxysection{Lin\+Alg\+Error Class Reference}
\hypertarget{classnumpy_1_1linalg_1_1linalg_1_1_lin_alg_error}{}\label{classnumpy_1_1linalg_1_1linalg_1_1_lin_alg_error}\index{LinAlgError@{LinAlgError}}
Inheritance diagram for Lin\+Alg\+Error\+:\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[height=2.000000cm]{classnumpy_1_1linalg_1_1linalg_1_1_lin_alg_error}
\end{center}
\end{figure}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Generic Python-exception-derived object raised by linalg functions.

General purpose exception class, derived from Python's exception.Exception
class, programmatically raised in linalg functions when a Linear
Algebra-related condition would prevent further correct execution of the
function.

Parameters
----------
None

Examples
--------
>>> from numpy import linalg as LA
>>> LA.inv(np.zeros((2,2)))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "...linalg.py", line 350,
    in inv return wrap(solve(a, identity(a.shape[0], dtype=a.dtype)))
  File "...linalg.py", line 249,
    in solve
    raise LinAlgError('Singular matrix')
numpy.linalg.LinAlgError: Singular matrix\end{DoxyVerb}
 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
fluent20.\+1.\+0/cortex/pylib/nexus/numpy/linalg/\mbox{\hyperlink{linalg_8py}{linalg.\+py}}\end{DoxyCompactItemize}
